luzer/tests/CMakeLists.txt

```diff
@@ -138,8 +138,11 @@ target_link_libraries(luac PRIVATE
 set_target_properties(luac PROPERTIES PREFIX "")
 target_compile_options(luac PRIVATE
   -fsanitize=fuzzer-no-link
+  -fsanitize=undefined
```

Comment:
Why do we need undefined sanitizer in tests?

Reply:
Because test is for symbols required by sanitizer. `luac` is a target for fuzzing built with ubsan, we should check that symbols required by ubsan are provided by our library libfuzzer+sanitizer.